src/store/modules/DocumentsRequiredModule.js

```diff
@@ -1,21 +1,12 @@
 import { VuexModule, Module, Action } from 'vuex-class-modules';
+import ConnectError from '@endpass/connect/error';
```

Comment:
`RequiredDocumentsModule` looks better

Reply:
why? i can see main context at first that this is `documents`

Reply:
Because component's name sounds weird

Reply:
when you searching you will search by context `document` at first and it will be easier to find by context.